_posts/en/2019-03-06-developers-italia-at-google-summer-of-code-2019.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This news has been substituted by the English one. I would suggest removing this file and *then* rebasing to avoid conflicts.

Reply:
Agree, rebasing.